cli/types/package.json

```diff
@@ -11,10 +11,8 @@
     "scripts": {
         "publish:dev": "tsc && cp package.json build/package.json && cd build && npm publish --dry-run",
         "publish:prod": "tsc && cp package.json build/package.json && cd build && npm publish",
-        "generate": "npm run types:proxy && npm run types:factory && npm run types:govec && npm version patch",
-        "types:proxy": "cosmwasm-ts-codegen generate --schema ../../contracts/proxy/schema --out ./contracts --name Proxy",
-        "types:factory": "cosmwasm-ts-codegen generate --schema ../../contracts/factory/schema --out ./contracts --name Factory",
-        "types:govec": "cosmwasm-ts-codegen generate --schema ../../contracts/govec/schema --out ./contracts --name Govec"
+        "generate": "npm run types && npm version patch",
```

Comment:
This is so much cleaner, exactly how I wanted to see this being implemented, good one ☝️